setup.cfg

```diff
@@ -87,11 +87,10 @@ test =
     composeml >= 0.8.0
     urllib3 >= 1.26.5
     pyarrow >= 3.0.0
-koalas = 
-    pyspark >=3.0.0, <3.2.0
-    koalas >=1.8.1, <=1.8.2; python_version < '3.10'
+spark = 
+    pyspark >=3.2.0, <=3.2.1
     pandas >=1.3.0, <1.4.0
```

Comment:
are we able to remove the pandas<1.4 constraint now?

Reply:
not yet  there are still breaking changes

Reply:
are we tracking that issue somewhere?

Reply:
it seems like it, opened #1951